functions/Find-DbaCommand.ps1

```diff
@@ -0,0 +1,154 @@
+Function Find-DbaCommand
+{
+<#
+.SYNOPSIS
+Finds dbatools commands searching through the inline help text
+
+.DESCRIPTION
+Finds dbatools commands searching through the inline help text, building a consolidated json index and querying it because Get-Help is too slow
+
+.PARAMETER Tag
+Finds all commands tagged with this tag
+
+.PARAMETER Author
+Finds all commands tagged with this author
+
+.PARAMETER Rebuild
+Rebuilds the index
+
+.PARAMETER Pattern
+Searches help for all commands in dbatools for the specified pattern and displays all results
+
+.PARAMETER Confirm
+Confirms overwrite of index
+
+.PARAMETER WhatIf
+Displays what would happen if the command is run
+
+.NOTES
+Tags: find
+Author: niphlod
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/Find-DbaCommand
+
+.EXAMPLE
+Find-DbaCommand "snapshot"
+
+For lazy typers: finds all commands searching the entire help for "snapshot"
+
+.EXAMPLE
+Find-DbaCommand -Pattern "snapshot"
+
+For rigorous typers: finds all commands searching the entire help for "snapshot"
+
+.EXAMPLE
+Find-DbaCommand -Tag copy
+
+Finds all commands tagged with "copy"
+
+.EXAMPLE
+Find-DbaCommand -Author "chrissy"
+
+Finds every command whose author contains our beloved "chrissy"
+
+.EXAMPLE
+Find-DbaCommand -Pattern "snapshot" -Rebuild
+
+Finds all commands searching the entire help for "snapshot", rebuilding the index (good for developers)
+#>
```

Comment:
Maybe you can add an example using multiple params, like:
Find-DbaCommand -Author JFK -Tag speeding